article/app/model/LiveBlogHelpers.scala

```diff
@@ -79,4 +82,38 @@ object LiveBlogHelpers {
     }.getOrElse(Right(NotFound))
 
   }
+
+  def blockTextJson(page: LiveBlogPage, number: Int): JsValue = {
+
+    case class TextBlock(
+      id: String,
+      title: Option[String],
+      publishedDateTime: Option[DateTime],
+      lastUpdatedDateTime: Option[DateTime],
+      body: String
+    )
+
+    implicit val dateToTimestampWrites = play.api.libs.json.JodaWrites.JodaDateTimeNumberWrites
+
+    implicit val blockWrites = (
```

Comment:
You don't need to define this explicitly, you can use the Play writes macro

Reply:
Good shout for that, I'd like to leave that for a follow up pr though I've done too much for one round and this is existing code that's been moved.